modules/programs/most.nix

```
@@ -0,0 +1,22 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.most;
+
+in {
+  options.programs.most = {
+    enable = mkEnableOption "Most, a powerful paging program";
+    sessionPager =
+      mkEnableOption "use Most as the session pager, e.g. for manpages";
+  };
+
+  config = mkIf cfg.enable {
+    home.packages = [ pkgs.most ];
+    home.sessionVariables = mkIf cfg.sessionPager { PAGER = "most"; };
```

Could use an absolute path

```suggestion
    home.sessionVariables = mkIf cfg.sessionPager { PAGER = "${pkgs.most}/bin/most"; };
```

(or

```suggestion
    home.sessionVariables.PAGER = "most";
```

if the above suggestion is accepted).